integration_test/test_compute_correct_metrics.py

```diff
@@ -0,0 +1,74 @@
+import sigopt.xgboost
+from sklearn.metrics import (
+  accuracy_score,
+  classification_report,
+  mean_absolute_error,
+  mean_squared_error
+)
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+import xgboost as xgb
+import numpy as np
+
+iris = datasets.load_iris()
+X = iris.data
+y = iris.target
+num_class = 3
+X_train, X_test, Y_train, Y_test = train_test_split(X, y, test_size=0.3)
+D_train = xgb.DMatrix(X_train, label=Y_train)
+D_test = xgb.DMatrix(X_test, label=Y_test)
+
+
+class TestComputeMetrics(object):
+  def test_compute_classification_metrics(self):
+    params = {
+      'eta': 0.3,
+      'num_class': num_class,
+      'objective': 'multi:softmax'
+    }
+    xgb_params = {
+      'params': params,
+      'dtrain': D_train,
+      'evals': [(D_test, 'test0')],
+    }
+    ctx = sigopt.xgboost.run(**xgb_params)
```

Comment:
If just test compute metrics, it is no need to train a model and predict.

Reply:
That is a fair point. I do think this test is somewhat unnecessary, but it does test the metric logging correctness. Whereas test/test_compute_metrics.py tests a few different edge cases when computing these metrics. The tests could possibly merged, thoughts @harvey-intel ?

Reply:
Yea, it's a good idea to consolidate these into the `TestXGBoost` in `integration_test/test_xgboost.py`.

Reply:
Okay, so then I will remove test/test_compute_metrics.py and move tests in here, and also remove the sklearn dependency in requirements-dev.txt, is that okay?

Reply:
Yea, I would consolidate everything here and `test_log_metric.py` under `TestXGBoost` in `integration_test/test_xgboost.py`.